sys/posix/sockets/posix_sockets.c

```diff
@@ -363,8 +361,8 @@ int accept(int socket, struct sockaddr *restrict address,
                     res = -1;
                     break;
                 }
-                _htons_port(port);  /* XXX: sin(6)_port is supposed to be network byte
-                                     *      order */
+                *port = htons(*port); /* XXX: sin(6)_port is supposed to be
```

Comment:
wouldn't it make sense to use `network_uint16_t` as type for `port` ? Since `port` is supposed to be in network byte order .. according to the comment.